pandas/core/ops.py

```diff
@@ -1023,21 +1020,22 @@ def na_op(x, y):
                 mask = notna(xrav) & notna(yrav)
                 xrav = xrav[mask]
 
-                # we may need to manually
-                # broadcast a 1 element array
                 if yrav.shape != mask.shape:
-                    yrav = np.empty(mask.shape, dtype=yrav.dtype)
-                    yrav.fill(yrav.item())
+                    # FIXME: GH#5284, GH#5035, GH#19448
```

Comment:
tests that hit this

Reply:
See #19421.  Only one test hits this, and only in py3:

```
s = Series([2, 3, 4, 5, 6, 7, 8, 9, datetime(2005, 1, 1)])
s[::2] = np.nan
d = DataFrame({'A': s})

with pytest.raises(ValueError):
    d.__and__(s, axis='columns')
```

Reply:
This is a change and not just a refactor.  As per the comments, it is horribly broken, and better to raise intentionally than accidentally have the `yrav.item()` raise a ValueError.